packages/utils/astring/src/sourcemaps.js

```diff
@@ -0,0 +1,53 @@
+const VLQ_SHIFT = 5;
```

Comment:
This file isn't being used by astring, maybe we should just remove it?

Reply:
oh, weird